### DISCUSSION TITLE
BeamWidth Filter

### DISCUSSION THREAD

**Discussion by AFA141** [mapsforge/mapsforge#1332] [Q&A] [answered]
Hello, what is the best way to create BeamWidth Filter on map? 
The information for creating the beamwidth filter is angle and distance.
Platform: Desktop

Example of BeamWidth Filter :
https://wiki.towercoverage.com/wiki/96/transmission-site-transmitter-info

**Comment by devemux86** [comment, marked as answer]
Old question: use the `drawArc` or `fillArc` method of the platform canvas (get it with the graphic factory).
Desktop example:
```java
Graphics2D graphics = AwtGraphicFactory.getGraphics(canvas);
graphics.fillArc(...);
```
